### ISSUE TITLE
using the "share" function to import files omits wiping them

### ISSUE THREAD

**Issue by owosoup** [hardcore-sushi/DroidFS#300]
When selecting a file and choosing "share", then selecting droidFS, it will let me choose where to deposit it and I can validate it. But once imported, the dialogue box only confirms that it's been done and it doesn't ask if I want to wipe the original files afterwards.
This works when I import files directly in droidFS, and it asks me if I want to wipe the files (which I do).

**Comment by hardcore-sushi** [comment]
Yes, it has not been implemented because most apps do not grant write access when sharing files, so DroidFS wouldn't be able to wipe them. That said, it may still be worth considering implementing this feature for specific cases. What app do you import files from?

**Comment by owosoup** [comment]
Oh, then don't mind this, I'm using the default, proprietary file manager on my realme 9. But I can see this work with open-source file browsers with willing devs! Guess I'll leave this open for those wondering.
